src/Symfony/Component/Security/Http/Tests/RememberMe/PersistentTokenBasedRememberMeServicesTest.php

```diff
@@ -338,4 +343,13 @@ protected function getProvider()
 
         return $provider;
     }
+
+    protected function generateHash(string $tokenValue): string
+    {
+        $service = $this->getService();
+        $r = new \ReflectionMethod($service, 'generateHash');
```

Comment:
It'd be more robust to compy/paste the implementation instead of using reflection:
that way, if the implementation diverges from the test, we'll see a red flag.

Reply:
Ok, updated. (i hardcoded the prefix in test instead of using the constant, for the same reason)